### ISSUE TITLE
How to compute the gflops of ResNest?

### ISSUE THREAD

**Issue by kevinbro96** [zhanghang1989/ResNeSt#114]
Hi, there! I want to compute the gflops of ResNest on my task, but I don't know the gflops computing rule of SplAtConv2d.
So can you share the code of the computing of gflops of ResNest?

**Comment by kevinbro96** [comment]
Hello?

**Comment by zhanghang1989** [comment]
We used MXNet implementation to calculate FLOPs. https://github.com/Ldpe2G/DeepLearningForFun/tree/master/MXNet-Python/CalculateFlopsTool

**Comment by kevinbro96** [comment]
The MXNet implementation only consider layers: Convolution, Deconvolution, FullyConnected, Pooling, relu. But the SplAtConv2d contains many split attention modules which are not included in the FLOPS calculation. So maybe the calculation results is not correct. The actual FLOPS  could be much larger than the paper reported.

**Comment by zhanghang1989** [comment]
Channel-wise multiplication is very efficient/cheap. The FLOPs is similar to SENet.

**Comment by kevinbro96** [comment]
I don't think so. The GFLOPS of channel-wise multipication is at least equal to the size of the output feature map, because each point in the feature map need to be multiplied by a value.

**Comment by zhanghang1989** [comment]
The computation of channel-wise multiplication is the same as SENet.

**Comment by kevinbro96** [comment]
I'm afraid not. I have tested the latency (time of forwarding a image) of SENet and ResNest. The latency of SENet101 is 1.2s, while that of ResNest101 is 2.2s. Although I don't know how to compute the FLOPs of SplAtConv2d, I think latency can represent FLOPS to some extent.
